udata/core/followers/api.py

```diff
@@ -46,7 +46,12 @@ def get(self, id):
     @api.doc(id='follow', description=NOTE)
     def post(self, id):
         '''Follow an object given its ID'''
-        follow, created = self.model.objects.get_or_create(follower=current_user.id, following=id, until=None)
+        try:
+            follow = self.model.objects.get(follower=current_user.id, following=id, until=None)
+            created = False
+        except self.model.DoesNotExist:
+            follow = self.model.objects.create(follower=current_user.id, following=id, until=None)
+            created = True
```

Comment:
Do we make it a reusable get_or_create here:
https://github.com/etalab/udata/blob/master/udata/models/__init__.py#L84-L91